plugin-server/tests/worker/vm.test.ts

```diff
@@ -1272,23 +1283,28 @@ describe('vm tests', () => {
             }
             await delay(1010)
 
-            expect(fetch).toHaveBeenCalledTimes(15)
+            expect(fetch).toHaveBeenCalledTimes(20)
```

Comment:
What is this change caused by?

Reply:
We stopped checking type errors on test files, so I noticed this file had a bunch and fixed them. One of them was that I added the missing `uuid` to all events. That causes the payload to be larger, making this test for buffering based on bytes flush more often. 

Also took me a while to figure this out, was very confused, super annoying.